spec/requests/providers_spec.rb

```diff
@@ -1791,6 +1791,52 @@ class DummyProvider; end
     end
   end
 
+  context 'configuration_profiles subcollection' do
+    let(:configuration_profile) { FactoryBot.create(:configuration_profile) }
+    let(:ems) { FactoryBot.create(:configuration_manager) }
+
+    context 'GET /api/providers/:id/configuration_profiles' do
+      it 'returns the lans with an appropriate role' do
+        api_basic_authorize(collection_action_identifier(:providers, :read, :get))
+
+        expected = {
+          'resources' => [{'href' => api_provider_configuration_profile_url(nil, ems, configuration_profile)}]
+        }
+        get(api_provider_configuration_profiles_url(nil, ems))
+
+        expect(response).to have_http_status(:ok)
+        expect(response.parsed_body).to include(expected)
+      end
+
+      it 'does not return the lans without an appropriate role' do
+        api_basic_authorize
+
+        get(api_provider_configuration_profiles_url(nil, ems))
+
+        expect(response).to have_http_status(:forbidden)
+      end
+    end
+
+    context 'GET /api/providers/:id/lans/:s_id' do
```

Comment:
probably didn't mean to add this, now a dup.

Reply:
Fixed